oak/server/loader/oak_runner_main.cc

```diff
@@ -28,6 +29,8 @@
 ABSL_FLAG(std::string, application, "", "Application configuration file");
 ABSL_FLAG(std::string, ca_cert, "", "Path to the PEM-encoded CA root certificate");
 ABSL_FLAG(std::string, private_key, "", "Path to the private key");
+ABSL_FLAG(std::string, private_key_env, "OAK_TLS_PRIVATE_KEY",
+          "Name of environment variable providing the path to the private key");
```

Comment:
This seems... a bit convoluted :) I think the problem with the current config is that we don't pass flags to the binary from the k8s config, but I think we should, rather than setting them in the docker image. In fact I am thinking that the docker image should probably just be for the oak_loader binary only (i.e. the same for any application), and even the application configuration and any other files and flags should just be specified in `pod.yaml`. WDYT? cc @ipetr0v

Reply:
On the first point: it is convoluted because `entrypoint` to a docker image does not expand environment variables. I will test specifying the command in `pod.yaml` and removing the entry point form the docker image.

Another option is to always specify a fixed path and then mount different volumes depending on where it is run (eg mount secrets for k8s or local directory for local docker).

Regarding the second part, I am not sure what you mean with 'any other files'. `pod.yaml` does not provision files into the image. It just sets configuration. If the app config and the certificates are not already inside the container image it would need to be downloaded from somewhere.

Reply:
Ah yes, you are right. Perhaps we should have a Docker image with the loader + app config at some known location, and then specify all the flags from `pod.yaml` (which will always include the flag to specify the app config from the known location, plus possibly other flags).

Reply:
Removed `entrypoint`s from docker images and added `command`s and `args` to `pod.yaml`. Reverted c++ changes.